package.json

```diff
@@ -61,6 +61,7 @@
   },
   "devDependencies": {
     "@changesets/cli": "2.26.2",
+    "@web-types/lit": "2.0.0-3",
```

Comment:
Unfortunately, there is no better way than adding a new package. This package is not often updated and is mainly made of JSON files. Providing that Intellij has quite some usage at the Post, I think it's reasonable to add this package.

Reply:
Yes good idea